src/Compilers/VisualBasic/Test/Symbol/CrossLanguageTest.vb

```diff
@@ -0,0 +1,52 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+
+    Public Class CrossLanguageTest
+        Inherits BasicTestBase
+
+        <Fact>
+        Public Sub CanAchieveHandledEvents()
+            Dim csharpSource = <![CDATA[
+[assembly:System.CLSCompliant(true)]
+public class Sample
+{
+    public void Create()
+    {
+    }
+}
+]]>
+
+            Dim csharpCompilation = CreateCSharpCompilation(
+                csharpSource,
+                compilationOptions:=New CSharp.CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary),
```

Comment:
>compilationOptions:=New CSharp.CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary), [](start = 16, length = 93)

Is this argument and those below necessary?

Reply:
@AlekseyTs, no, they don't. Removed them